tactics/class_tactics.ml

```diff
@@ -100,7 +100,7 @@ let _ =
 
 let _ =
   declare_bool_option
-    { optdepr  = false;
+    { optdepr  = true; (* remove in 8.8 *)
```

Comment:
@JasonGross you are right. Once again, I was fooled by the fact that this option (Typeclasses Limit Intros) is set in the 8.5 compatibility file for no good reason since it's on by default. Moreover, @mattam82 the documentation currently says "this option is on by default in 8.6 and below" which makes no sense in the 8.7 documentation. Was your intention to change its default in 8.7? In that case, the documentation could be changed to "this option is off by default since 8.7", it should be added to the 8.6 compatibility file and only deprecated in 8.8., to be removed in 8.9 as @JasonGross suggested.